src/models/analytics-panel.ts

```diff
@@ -59,6 +59,7 @@ const schema = Joi.object().keys({
   layout: layoutSchema.default([]),
   cards: Joi.array().items(Joi.object().keys({
     title: Joi.string().required().example('My widget'),
+    subTitle: Joi.string().allow('').optional(),
```

Comment:
actually `.allow('')` is redundant, while field is optional

Reply:
I don't think that's true:
![image](https://user-images.githubusercontent.com/1840153/147974730-3003a85f-5434-4183-9fd8-716a4580124e.png)

Reply:
Oh. My bad, closing, sorry for disturbing.